src/quo2/components/context_tags.cljs

```diff
@@ -46,16 +46,21 @@
                  :size :paragraph-2}
       (trim-public-key public-key)]]))
 
-(defn user-avatar-tag [_ _]
-  (fn [params public-key]
-    (let [{:keys [name photo]} @(re-frame/subscribe [:contacts/name-and-photo public-key])]
+(defn user-avatar-tag [params public-key photo contacts]
+  (fn [params public-key photo contacts]
```

Comment:
Hi, why do we need to pass all contacts here. why can't we just pass final name?

Reply:
Done

Reply:
Please check Parvesh